spec/features/houses/delete_house_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe "deleting a house", type: :feature do
+  let!(:house1) { FactoryBot.create :house }
```

Comment:
usually we don't use numbers unless we have more than one house... so you could use just `house` instead